docs/vsc_ugent.md

```diff
@@ -0,0 +1,37 @@
+# nf-core/configs: University of Ghent High Performance Computing Infrastructure (VSC)
+
+> **NB:** You will need an [account](https://www.ugent.be/hpc/en/access/faq/access) to use the HPC cluster to run the pipeline.
+
+First you should go to the cluster you want to run the pipeline on. You can check what clusters have the most free space on this [link](https://shieldon.ugent.be:8083/pbsmon-web-users/). Use the following commands to easily switch between clusters:
+
+```shell
+module purge
+module swap cluster/<CLUSTER>
+```
+
+Before running the pipeline you will need to create a PBS script to submit as a job.
```

Comment:
Any reason to suggest doing this as running a bash script, instead of using `.bashrc` etc? Just out of curiosity?

Reply:
Do you mean the `module swap` command?

Reply:
Yeah so on my old HPC I had a different `.bashrc` on each cluster t hat did `module load nextflow` and `export NXF_SINGULARITY_CACHEDIR=xxx` so I never had to think about it

Reply:
Yeah true that's useful :) I'll think about it, but maybe it's a bit easier for other users to just add this to the PBS script

Reply:
Yeah if you're using a PBS script to launch anyway then I guess it's good. We used to run Nextflow on the head node (detached with `-bg` flag / in `tmux` etc), so yeah that's a key difference 👍🏻